core/tests/protractor_utils/ProfilePage.js

```diff
@@ -97,10 +113,14 @@ var ProfilePage = function() {
       throw new Error(
         'There is no exploration card with name ' + explorationName);
     }
+    await waitFor.visibilityOf(explorationsCardByName,
+      'explorationsCardByName label element taking too long to appear');
```

Comment:
This isn't needed because we've already interacted with all these elements. They wouldn't have disappeared from when we used them earlier in the function